_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,140 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Forseti Notifier can send a variety of notifications to alert you
+of Forseti events. These notifications can be configured to be sent
+to different channels, and in different formats.
+
+### Notification Types
+
+  1. Inventory summary
+  1. Scanner summary
+  1. Violations
+  1. CSCC findings
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. GCS
```

Comment:
Branding: Cloud Storage

Reply:
Done, fixed everywhere, execept where used in the code & config.